completions/ip

```diff
@@ -456,7 +456,21 @@ _comp_cmd_ip()
                     # TODO
                     ;;
                 show)
-                    # TODO
+                    case $prev in
+                        dev)
+                            _comp_compgen_available_interfaces
+                            ;;
+                        name)
+                            _comp_compgen_split -- "$(
+                                {
+                                    ip -c=never ntable show 2>/dev/null || ip ntable show
+                                } | sed -e '/inet.*/!d' -e 's/inet6\{0,1\} //g'
```

Comment:
```suggestion
                                } | sed -ne 's/^inet6\{0,1\} //p'
```

`/inet.*/` and `/inet6\{0,1\} /` match `inet ` in the middle of a line (such as `    dev clarinet ...`). If it's supposed to match at the beginning of line, the flag `g` is unnecessary.

Reply:
The joining of the sed commands is clever, changed according to the suggestions. Also added `command` before the sed as it is elsewhere

Reply:
Thanks!